### PR TITLE
Logging Release 1.5.0

### LINKED ISSUE #4861: Logging: Release new version of google-cloud-logging
Hello,

The latest version of Google cloud logging is https://pypi.org/project/google-cloud-logging/#history, in October 2017. We wanted a PyPi release with the latest changes with respect to creating sinks with `unique_writer_identity` set. 

We really would prefer not using the latest google-cloud module as we want to use different versions of various google cloud services.

Thanks,
Sneha

### PR DESCRIPTION
**Pull request by chemelnucfin**
Closes #4861

### CODE REVIEW

File: logging/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,29 @@
 
 [1]: https://pypi.org/project/google-cloud-logging/#history
 
+## 1.4.1
+
+### Implementation Changes
+  - Logging pass in grace period and batch size to workers (#4369)
```
Comment (jonparrott) [inline logging/CHANGELOG.md]: This doesn't read well. Can we re-phrase this a bit?

File: logging/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,29 @@
...
+  - Unique Writer Identity (#4595, #4704, #4706, #4708)
```
Comment (jonparrott) [inline logging/CHANGELOG.md]: This is not a complete sentence.

File: logging/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,29 @@
...
+  - Making Sink.filter Optional (#4699)
```
Comment (jonparrott) [inline logging/CHANGELOG.md]: Made it optional in what?

File: logging/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,29 @@
...
+  - Capture Server-generated writerIdentity during LogSink.create (#4707)
```
Comment (jonparrott) [inline logging/CHANGELOG.md]: Rephrase to use active voice and make LogSink.create the subject:

```
Made `LogSink.create` capture the server-generated `writerIdentity`.
```

File: logging/CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,29 @@
...
+  - Generate GAPIC and Remove GAX (#4759)
```
Comment (jonparrott) [inline logging/CHANGELOG.md]: This should be stated as "The underlying client library was re-generated to pick up new features and bugfixes."

File: logging/CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,29 @@
...
+  - Move code path of get_gae_labels() to emit() (#4824)
```
Comment (jonparrott) [inline logging/CHANGELOG.md]: Rephrase:

```
Changed `AppEngineHandler` to apply labels from `get_gae_labels()` during `emit()`.
```

File: logging/CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,29 @@
...
+
+### Dependencies
+  - Pinning `Django` test dependency to < 2.0 in Python 2.7 (#4519)
+
+### Tests
```
Comment (jonparrott) [inline logging/CHANGELOG.md]: You can drop this section.

File: logging/CHANGELOG.md [hunk 8]
```diff
@@ -4,6 +4,29 @@
...
+  - Test for retrieve_metadata_server against transparent DNS proxies (#4698)
+  - Adding cleanup for Pub/Sub topic in logging system test (#4532)
+
+### Minor Bug Fixes
```
Comment (jonparrott) [inline logging/CHANGELOG.md]: Either these should have a one-sentence summary of what they fix or they should be omitted.

**Comment by chemelnucfin** [comment]
@jonparrott PTAL, thanks.

**Comment by jonparrott** [comment]
@chemelnucfin this is certainly not a patch release, this should be v1.5.0. Can we wait until #4762 is merged and rebase?

**Comment by tcwalther** [comment]
Would be absolutely wonderful if this was released soon :-)

**Comment by chemelnucfin** [comment]
PTAL at your convenience.

**Pull request closed without merging**